src/components/Editor/index.js

```diff
@@ -28,52 +28,50 @@ const styles = {
     }
 };
 
-const Editor = ({ modules, actions }) => {
-    return (
-        <div style={styles.editor}>
-            {modules.map((module, key) => (
-                <textarea
-                    rows={1}
-                    ref={el => 
-                        el && el.scrollHeight !== module.height ?
-                            actions.module.update({
-                                id: module.id,
-                                height: el.scrollHeight,
-                            }) :
-                            el
-                    }
-                    onChange={e =>
-                        e.currentTarget.value ?
-                            actions.module.update({
-                                id: module.id,
-                                value: e.currentTarget.value,
-                                height: 0
-                            }) :
+const Editor = ({ modules, actions }) => (
+    <div style={styles.editor}>
+        {modules.map((module, key) => (
+            <textarea
```

Comment:
Do we want the `textarea` to be tied to the `Editor` or are we wanting `textarea` to be just a view that `Editor` displays?

I.E: Should `textarea` be its own component outside of `Editor` or do we want to keep it defined here?